app/isomorphic/components/login/AccountModal/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { func } from "prop-types";
+
+import { Modal } from "../Modal";
+import { Login } from "../../molecules/forms/login";
+import { SignUp } from "../../molecules/forms/sign-up";
+import { OTP } from "../../molecules/forms/otp";
+import { ForgotPassword } from "../../molecules/forms/forgot-password";
+
+import "./account-modal.m.css";
+
+const AccountModal = ({ onBackdropClick, checkForMemberUpdated }) => {
+  const [activeTab, setActiveTab] = useState("login");
+  const [member, setMember] = useState(null);
+  const [otpToken, setOtpToken] = useState(null);
+  // const [error, setError] = useState({});
+
+  const otpHandler = (member, otpDetails) => {
+    setMember(member);
+    setOtpToken(otpDetails["email-token"]);
+    setActiveTab("otp");
+  };
+
+  const onSuccess = member => {
+    console.log("need to get the verify email api from BK library");
+    // verifyEmail(member.email)
+    //   .then(res => otpHandler(member, res))
+    //   .catch(error => setError(error));
```

Comment:
remove  commented code if it's not required from all the places

Reply:
required in future when functionality is implemented